### PR TITLE
[KULRICE-14144] Spring update

### PR DESCRIPTION
**Pull request by bsmith83**

### CODE REVIEW

**Comment by coreypedersen** [comment]
Was another jira created to add the Oracle SQL upgrades scripts?

**Comment by bsmith83** [comment]
No the other jira was not created (as far as I know), can you create one?

**Comment by cniesen** [comment]
retest this please

**Comment by cniesen** [comment]
There are a few test that are failing. 
We also need the oracle scripts otherwise our deploy and aft ci jobs will fail since some of them use an oracle database.

**Comment by jruch** [comment]
looks good.

**Comment by shahess** [comment]
Brian, 

I attached an updated oracle script to KULRICE-14144 along with some notes.  As long as Corey doesn't see any issues with the change I made, the updated oracle script is ready to be brought into the pull request.  

Thanks,
Shannon

**Comment by bsmith83** [comment]
DB script in

File: db/sql/src/main/resources/org/kuali/rice/rice-sql/upgrades/2.5.0/mysql/server/bootstrap/2015-02-05--KULRICE-14144.sql [hunk 1]
```diff
@@ -0,0 +1,152 @@
+--
```
Comment (cniesen) [inline db/sql/src/main/resources/org/kuali/rice/rice-sql/upgrades/2.5.0/mysql/server/bootstrap/2015-02-05--KULRICE-14144.sql]: This file should be in the 2.6.0 folder.

File: db/sql/src/main/resources/org/kuali/rice/rice-sql/upgrades/2.5.0/oracle/server/bootstrap/2015-03-10--KULRICE-14144.sql [hunk 2]
```diff
@@ -0,0 +1,102 @@
+-- KULRICE-14144 changes for SpringFramework 4.1.4 with Quartz 2.2.1
```
Comment (cniesen) [inline db/sql/src/main/resources/org/kuali/rice/rice-sql/upgrades/2.5.0/oracle/server/bootstrap/2015-03-10--KULRICE-14144.sql]: This file should be in the 2.6.0 folder.

**Comment by cniesen** [comment]
Looks good

**Comment by coreypedersen** [comment]
Other than my last comment looks ok to me.

**Comment by cniesen** [comment]
From Brian:
No, override="false" means it's not overriding the setting if one exists already.

**Pull request merged**